application/src/main/resources/application.yml

```diff
@@ -28,11 +28,23 @@ management:
 # Security Config
 #server:
 #  ssl:
+# SSL configuration using certificate stores
 #    enabled: false
 #    key-store: <path to keystore>
 #    key-store-password: <password>
 #    key-store-type: <keystore type>
 #    key-alias: <key alias>
 #    key-password: <key password>
-# Rate at which certificate are reloaded automatically
+#    client-auth: false
+#    protocol: <protocol>
+#    ciphers: <ciphers>
+# SSL configuration using certificates in PEM format
+#    enabled-certificate: false
+#    certificate: <path to certificate>
```

Comment:
This looks very strange with having duplicated fields.
Can't you use the same approach as in CQL case? I.e if PEM fields are specified, use PEM. If not use keystore.